src/main/resources/messages/CoderGatewayBundle.properties

```diff
@@ -93,3 +93,21 @@ gateway.connector.settings.header-command.comment=An external command that \
   outputs additional HTTP headers added to all requests. The command must \
   output each header as `key=value` on its own line. The following \
   environment variables will be available to the process: CODER_URL.
+gateway.connector.settings.tls-cert-path.title=Cert Path:
```

Comment:
Not really important, but the other settings are `Capitalized like this:` so changing these to `Cert path:` etc might be good for a more consistent look (or change the capitalization of the others).